src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -909,7 +1041,7 @@ std::shared_ptr<ov::npuw::LLMCompiledModel> ov::npuw::LLMCompiledModel::import_m
 
     if (!encrypted) {
         EncryptContext ctx(false, nullptr, nullptr);
-        auto compiled_model = ov::npuw::LLMCompiledModel::deserialize(stream, plugin, properties, ctx);
+        auto compiled_model = ov::npuw::LLMCompiledModel::deserialize(stream, plugin, properties, ctx, contains_3_models);
```

Comment:
Let LLMCompiledModel itself decide how much models is there. Please don't do abstraction leaks like this.

Reply:
Thanks, fixed!